sdk/identity/identity/src/msal/nodeFlows/msalOpenBrowser.ts

```diff
@@ -96,7 +106,13 @@ export class MsalOpenBrowser extends MsalNode {
           codeVerifier: this.pkceCodes?.verifier,
         };
 
-        this.acquireTokenByCode(tokenRequest)
+        const acquire = hasNativeBroker() ? this.acquireTokenInteractive.bind(this, {
```

Comment:
I'm a bit confused why you are building logic around whether or not there is a native broker. This is an implementation detail of msal-node and should be mostly opaque to you. You should simply call acquireTokenSilent or acquireTokenInteractive and let msal-node do the right thing.

Reply:
`hasNativeBroker()` indicates whether the user has indicated that they want to import the azure sdk's native broker plugin and based on that we will pass the info received from the user to the msal